01-Login/README.md

```diff
@@ -86,9 +55,9 @@ This will then launch your app in the local emulation environment for the platfo
 
 ## Update Auth0 Dashboard Configuration
 
-1. After executing the `run` command, your `config.xml` file should contain `<allow-navigation>` tag(s).
-2. Make note of IP address URLs from any `<allow-navigation>` tags.
-3. Log into your [Auth0 Dashboard](https://manage.auth0.com) and go to your Ionic app client's **Settings**.
-4. Add the full allowed navigation addresses (including `http://` and ports) to the **Allowed Origins (CORS)** settings for your Auth0 client and click the **Save Changes** button.
+1.  After executing the `run` command, your `config.xml` file should contain `<allow-navigation>` tag(s).
+2.  Make note of IP address URLs from any `<allow-navigation>` tags.
```

Comment:
IPs should not be needed for this, Ionic runs the app on localhost:8080 that should suffice.